docs/website/configuration.rst

```diff
@@ -0,0 +1,120 @@
+
+Environment Configuration Instruction
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Next, I will give a step by step instruction on how to compile this website from scratch.
+
+1. Converting md files into ipynb files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1.1 Using existing ipynb from the website
+:::::::::::::::::::::::::::::::::::::::::
+
+If you don’t modify the examples section, what you need to do here is not converting the md files manually(which is time-consuming), but to get the ipynb files that are already ready for use. They can be found by clicking on the “DOWNLOAD THIS TUTORIAL” button on each page of the examples, such as `the word embedding tutorial <https://gluon-nlp.mxnet.io/master/examples/word_embedding/word_embedding.html>`__. After downloading these files, please put them in the same folder along with each md file. And then *delete* the md files. After these steps, you can skip the remaining of this section and go to the next section.
+
+1.2 Generating new ipynb from md files
+:::::::::::::::::::::::::::::::::::::::::
+
+If you need to modify the examples, then you will need a server to do all the calculation because converting a ``md`` file into a ``ipynb`` file means running all the scripts in it and then save all the outputs.
+
+To accomplish this task, we recommend you to use the instance ``g4dn.xlarge`` on Amazon EC2. For convenience, you can search *deep learning* in the filter bar to select the deep learning-enabled machines, where you will have no need of installing addition drivers.
+
+After you have got the machine and logged to the machine, you will see that on the welcome screen, it shows the conda environments if you have selected a deep learning machine. Here we will choose the MXNet one, use the command below:
+
+.. code:: bash
+
+    source activate mxnet_p36
+
+We will use the commands below to update the packages on this machine
+
+.. code:: bash
+
+    sudo apt-get update
+    sudo apt-get upgrade
+
+Use the commands below to install or update necessaryy packages:
+
+.. code:: bash
+
+    pip install notedown
+    pip install --upgrade sphinx
+    pip install --user -U numpy
+    pip install "nltk==3.2.5"
+
+After installing the packages above, get the GluonNLP repository from GitHub via the command:
+
+.. code:: bash
+
+    git clone https://github.com/dmlc/gluon-nlp
+
+Then navigate to the gluon-nlp folder
+
+.. code:: bash
+
+    cd gluon-nlp
+
+Use the command below to install the necessary packages in python:
+
+.. code:: bash
+
+    pip install --user -e '.[dev]'
+
+If necessary, you might still need to configure the packages like below:
+
+Use ``python`` command to get into the python execution screen, and then type the commands below to install the necessary packages inside python:
+
+.. code:: bash
+
+    import nltk
+    nltk.download('perluniprops')
+    nltk.download('nonbreaking_prefixes')
+    nltk.download('punkt')
+
```

Comment:
You can replace this by
`python3 -m nltk.downloader perluniprops nonbreaking_prefixes punkt`

Overall, I think you can reduce the amount of text and codeblocks here. 
In the end users should only need to run

```
pip install --user -e '.[extras,dev]'
python3 -m nltk.downloader perluniprops nonbreaking_prefixes punkt
```